AutomatedReview/BuildFailureRCA.md

```diff
@@ -0,0 +1,78 @@
+# Build Failure RCA Guide
+
+Self-Service build failure root cause analysis (RCA) system helps us identify and categorize root causes of Jenkins builds failures by scanning build log and searching for certain log patterns. Build failure RCA patterns are stored in O3DE Github repository and are maintained by all O3DE contributors.
+
+### Workflow
+![Workflow](./images/build_failure_rca1.png)
```

Comment:
I think we can take out the bottom half of this diagram for the fork Jenkins setup, since that section is only relevant to internal users.  The rest of the diagram looks good though.

Reply:
Updated, thanks!